extension/addreactionparticipants.js

```diff
@@ -0,0 +1,60 @@
+const addReactionParticipants = currentUser => {
+	$('.comment-reactions').each((index, reactionsContainer) => {
```

Comment:
This would require refactoring the initial selection of the container, but I don't see a benefit unless we pass in the new selection on the click. I don't know that we'll see a big improvement, and there's a positive side effect that if any of the _other_ comments are out of date (because of a socket) they'll get refreshed. I would do this in tandem with the socket thing.